packages/tools/src/utilities/math/line/distanceToPointSquaredInfo.ts

```diff
@@ -0,0 +1,41 @@
+import type { Types } from '@cornerstonejs/core';
+import distanceToPointSquared from '../point/distanceToPointSquared';
+
+export default function distanceToPointSquaredInfo(
+  lineStart: Types.Point2,
+  lineEnd: Types.Point2,
+  point: Types.Point2
+): {
+  point: Types.Point2;
+  distanceSquared: number;
+} {
+  let closestPoint: Types.Point2;
+  const d2 = distanceToPointSquared(lineStart, lineEnd);
+
+  if (d2 === 0) {
+    closestPoint = lineStart;
+  }
```

Comment:
not sure what is happening here, you are using the distance of line to point, but passing both line start and end?

Is this because you want to check if they are the same?

How about simple 

```js
const areEqual = lineStart[0] === lineEnd[0] && // etc.
```